pkg/stanza/operator/helper/emitter.go

```diff
@@ -17,14 +17,14 @@ import (
 // LogEmitter is a stanza operator that emits log entries to a channel
```

Comment:
No longer to a channel, perhaps this?
```suggestion
// LogEmitter is a stanza operator that emits log entries to the consumer callback function `consumerFunc`
```